README.md

```diff
@@ -643,8 +643,8 @@ A simple example of how the package can be used looks like this:
 ```javascript
 import React from "react";
 import ReactDOM from "react-dom";
-import { AddToCheckListEntry } from "@danskernesdigitalebibliotek/ddb-react";
-import "@danskernesdigitalebibliotek/ddb-react/components.css";
+import { AddToCheckListEntry } from "@danskernesdigitalebibliotek/dpl-react";
```

Comment:
Actually there was only that one. So I have deleted that.
But the README.md should have some more iterations.
I think it would make sense to update it as we go along.
And look at it again as a part of the post-project cleanup.

Reply:
And I deleted the NPM section of the README since we moved away from working with that.